### DISCUSSION TITLE
Area vs Arc

### DISCUSSION THREAD

**Discussion by Erlicht** [FSchwendinger/EffectVisR#3] [Ideas]
Given the potentially difficult interpretation of areas, an option to show just the arc for the confidence intervall would be nice. So just draw the dots in the last x rows before the border instead of from the origin to the border.

**Comment by FSchwendinger** [comment]
- [X] Addressed by including the `area_arc` argument, allowing the user to specify the size of the fan. 0 produces a complete fan. E.g. 0.95 creates an arc.
